platform-sdk/swirlds-state-api/src/main/java/com/swirlds/state/lifecycle/info/NodeInfo.java

```diff
@@ -39,7 +39,7 @@ public interface NodeInfo {
      * @return whether this node has zero stake.
      */
     default boolean zeroStake() {
```

Comment:
The term was changes to weight, but this method says stake.

Reply:
Ah yes, changed to `zeroWeight()`.